### ISSUE TITLE
Trigger configuration

### ISSUE THREAD

**Issue by vasfvitor** [Adsmurai-Google-Tag-Manager-Templates/adsmurai-facebook-conversions-api-tag#5]
Hi

Just to make sure we are in the same line of thought:
(I'm using code format to reference long GTM stuff)
In the Server GTM container, I have the client: `adsmurai-facebook-conversions-api-client` named as cAPI and the tag `adsmurai-facebook-conversions-api-tag`.

The client configuration is OK.

But it's about the tag I have a question: right now I'm using it with the trigger `Client Name equals cAPI` (that is, the client is the trigger). I believe it's working because I've been receiving both events from server and browser at Facebook Event Manager, as it should. 

My question is if this is the correct trigger. I've seen people using the trigger `Event Name matches Regex .+`. Facebook default/automatic implementation via GTM have it too but it was a mess of tags last time (long time) I tried.

So, I'm not sure the trigger is based on implementation (I mean, maybe with adsmurai-api-tag the trigger is one but with others tag may be different) or if there's a default trigger that should work regardless of tag.

To sum up: What's the correct trigger for this tag at the Server container?

Thanks

**Comment by angelbt91** [comment]
Hi @vasfvitor ! :)

I guess there are many triggers that would work ok for our use case. But I'm always using the same one in all my CAPI implementations, as I find it the most straight-forward solution, and it's the same one you are using: the client name.

So, I always name my client "Facebook CAPI client" and in my tags, I use a trigger configured as "Client name equals `Facebook CAPI client`". This way the tag fires every time the client sends an event expecting a tag to handle it.

Let me know if this helped you, or if you have any doubt left.

Thanks!

**Comment by vasfvitor** [comment]
Hi, I got it. Makes sense now.

> the tag fires every time the client sends an event expecting a tag to handle it

Now I understand why this works, I was looking through the GTM Server debug and I notice there was a correlation between tags and client, but I didn't give it a thought, thanks again.
